DEVELOPERS.md

```diff
@@ -44,9 +36,8 @@ apt-get install -y gnupg && \
   ACCEPT_EULA=Y apt-get install -y mssql-tools
 export PATH=$PATH:/opt/mssql-tools/bin
 
-# java
-apt-get update && apt-get install -y default-jre
 ```
+* Set up virtual env: `make devenv`
```

Comment:
Might be good to mention this only supports linux because of the wheel file specified in requirements.prod.txt?

Reply:
done